packages/react-ui/internal/themes/DefaultTheme.ts

```diff
@@ -678,9 +678,12 @@ export class DefaultTheme {
   public static calendarCellActiveHoverColor = 'white';
   public static calendarCellWeekendColor = '#f00';
   public static calendarCellTodayBorder = '1px solid #8b8b8b';
+  public static calendarCellBorderRadius = '50%';
   public static calendarCellSelectedBgColor = '#e9e9e9';
   public static calendarCellSelectedFontColor = 'inherit';
-  public static calendarCellSize = '30px';
+  public static calendarCellHeight = '30px';
+  public static calendarCellWidth = '30px';
```

Comment:
```suggestion
  public static calendarCellSize = '30px'; // deprecated
  public static get calendarCellHeight() {
    return this.calendarCellSize;
  }
  public static get calendarCellWidth() {
    return this.calendarCellSize;
  }
```

Поскольку список переменных темы является частью публичного апи, мы не можем их удалять в немажорном релизе без обратной совместимости. 

Можно пока вернуть старую переменную, пометить ее `deprecated`, а новые временно сделать на основе старой. В мажорном релизе отрефакторим.

Reply:
Понял проблему, не подумал об этом. Поправил.